shale/src/disk_address.rs

```diff
@@ -159,13 +160,9 @@ impl std::ops::BitAnd<usize> for DiskAddress {
     }
 }
 
-impl DiskAddress {
-    const MSIZE: u64 = 8;
```

Comment:
```rust
const MSIZE: u64 = size_of::<Self>() as u64;
```
Then, you can leave the rest of the code as it was. 

You still want the three places where this was previously used to be the same value, so it doesn't make sense to call the same function three different times.